src/main/java/com/google/cloud/spanner/jdbc/JdbcPreparedStatement.java

```diff
@@ -1,136 +0,0 @@
-/*
```

Comment:
Please do not remove this file as it might be needed for SpannerToGCS template

Reply:
This file is not referenced by SpannerToGCS at all. And I verified that SpannerToGCS can run without it with the dialect registered.

Reply:
It is called internally by spark when it uses Spanner JDBC driver.
We had to override this class from its jar file https://github.com/googleapis/java-spanner-jdbc/blob/main/src/main/java/com/google/cloud/spanner/jdbc/JdbcPreparedStatement.java

It is needed by the other template "Spanner To GCS"